recipes/julia/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: julia
+  version: "0.5.2"
+
+source:
+  fn: v0.5.2.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v0.5.2.tar.gz
+  sha256: efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7
+  patches:
+  - rpath.patch
+
+build:
+  skip: True  # [osx]
```

Comment:
Skip windows builds too? (ie:)

```
build:
  skip: True  # [py3k or osx or win]
```

The AppVeyor CI build fails with message:

```
Packages missing in current win-32 channels: 
  - autoconf 2.69 pl5.*
  - automake 1.15 pl5.*
  - openblas
  - gmp 0.5.*
```
...and 

```
Packages missing in current win-64 channels: 
  - autoconf 2.69 pl5.*
  - automake 1.15 pl5.*
  - openblas
  - gmp 0.5.*
```

(I realized the gmp version is wrong anyways, but not sure about the others)